src/main/webapp/views/editCard.html

```diff
@@ -15,104 +15,109 @@
               <hr>
               <div class="row mt-4">
                 <div class="col-lg-3">
                   <h5>Current business photo</h5>
                   <img id="business-picture" src="images/placeholderImage.png" class="card-img-top mt-2"
-                    alt="images/placeholderImage.png">
-                  <label class="mt-2" for="business-photo"><b>Select a new business photo</b></label>
-                  <input type="file" class="form-control-file" id="business-photo">
+                       alt="images/placeholderImage.png">
+                  <label class="mt-2" for="imageFile"><b>Select a new business photo</b></label>
+                  <input type="hidden" id="blobKey" name="blobKey">
+                  <input type="file" class="form-control-file" id="imageFile" name="imageFile">
                 </div>
                 <div class="col">
                   <p>Business Working Hours</p>
                   <div class="row">
                     <div class="col">
                       <p>From</p>
-                      <input type="time" class="form-control" id="business-startTime"
-                        aria-describedby="Business opened from" placeholder="From">
+                      <input type="time" class="form-control" id="startTime" name="startTime"
+                             aria-describedby="Business opened from" placeholder="From">
                     </div>
                     <div class="col">
                       <p>To</p>
-                      <input type="time" class="form-control" id="business-endTime"
-                        aria-describedby="Business opened to" placeholder="To">
+                      <input type="time" class="form-control" id="endTime" name="endTime"
+                             aria-describedby="Business opened to" placeholder="To">
                     </div>
                   </div>
                   <div class="row mt-4">
                     <div class="col-auto mt-3">
-                      <input class="form-control" type="checkbox" value="" id="delivery">
-                      <label for="delivery"><b>Delivery Service</b></label>
+                      <input class="form-control" type="checkbox" value="" id="hasDelivery" name="hasDelivery"
+                             onchange="updateDeliverySelection()">
+                      <label for="hasDelivery"><b>Delivery Service</b></label>
                     </div>
                     <div class="col">
                       <p>Delivery service distance in meters</p>
-                      <input type="number" class="form-control" id="business-distanceOfDelivery"
-                        aria-describedby="Business delivery distance" placeholder="Distance" value="1000">
+                      <input type="number" class="form-control" id="radius" name="radius"
+                             aria-describedby="Business delivery distance" placeholder="Distance" onchange="drawMap(getVendorInfo())">
```

Comment:
The value of `aria-describedby` needs to be an id.
[More info about it here.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)